src/js/components/Header.js

```diff
@@ -429,7 +430,7 @@ class Header extends React.PureComponent {
   }
 }
 
-const mapStateToProps = ({}) => ({});
+const mapStateToProps = ({ }) => ({});
```

Comment:
Is this needed?

Reply:
Yes, its needed below when exporting the component, this code was already there, I didn't change it.